Makefile

```diff
@@ -91,7 +98,25 @@ $(PDF):
 	ls -l $@
 
 $(FSM): bin
-	PG_AUTOCTL_DEBUG=1 ./src/bin/pg_autoctl/pg_autoctl do fsm gv | dot -Tpng > $@
+	$(PG_AUTOCTL) do fsm gv | dot -Tpng > $@
+
+$(TMUX_SCRIPT): bin
+	mkdir -p $(TMUX_TOP_DIR)
+	$(PG_AUTOCTL) do tmux script \
+         --root $(TMUX_TOP_DIR)  \
+         --nodes $(NODES)        \
+         --layout $(TMUX_LAYOUT) > $@
+
+tmux-script: $(TMUX_SCRIPT) ;
+
+tmux-clean:
+	pkill pg_autoctl || true
+	rm -rf $(TMUX_TOP_DIR)
+
+cluster: install tmux-clean $(TMUX_SCRIPT)
```

Comment:
Because `tmux-clean` and `$(TMUX_SCRIPT)` are targets that don't depend on eachother they are executed in parallel when you do:
```
make cluster -j9
```

This triggers errors like:
```
tmux start-server \; source-file ./tmux/script.tmux
./tmux/script.tmux: No such file or directory
```

I think it's best to simply call `make tmux-clean` and `make tmux-script` in the `cluster` target, instead of making it make dependencies. That way the order in which they are executed is clear.

Reply:
I see. Fixed now!